image/cinc/install_ubuntu_pkg.sh

```diff
@@ -64,3 +65,24 @@ systemctl mask \
 mkdir -pv /bin
 /tmp/generate_dhclient_script_for_fullstack.sh / || \
     echo "Failed to generate dhclient script for fullstack!"
+
+# Patch perf script.
+# On Ubuntu, '/usr/bin/perf' is a shell script that tries to find perf binary
+# that exactly matches current running kernel. This will break if
+# ovn-fake-multinode container runs on a host with different kernel than the
+# host that built the image.
+# It's no ideal, but since the perf should be backwards compatible, we can
+# circumvent this by linking '/usr/bin/perf' directly to an available
+# binary in '/usr/lib/linux-tools-<version>/perf'
+PERF_PATH=""
+
+for PERF_BIN in $(ls /usr/lib/linux-tools-*/perf); do
```

Comment:
My thought process was that since the `ls` output is sorted by name, the last iteration of the loop would point to the latest iperf/kernel version and that's the one that would be used. This is of course flimsy assumption and in some corner cases the "last" in the list may not necessarily be the latest kernel version, but imo it should hold true more often than not. And even in such corner cases, I don't think it would cause problems.

Reply:
Understood. Make sense.

Reply:
fwiw; `ls` sorts alphabetically by when no sort order is specified, so this would give the deterministic result of always pointing to the most recent version